selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/social/SocialLocator.java

```diff
@@ -1133,8 +1133,8 @@
   public static final By              ELEMENT_EDIT_MY_PROFILE_LINK                                      =
                                                                    By.xpath("//*[@id=\"UIUserNavigationPortlet\"]/div[1]/div/div/ul/li/a");
 
-  public static final By              ELEMENT_EDIT_MY_PROFILE_BUTTON                                    =
-                                                                     By.xpath("//*[@id='UIExperienceProfilePortlet']//*[@class='uiIconEdit uiIconLightGray']");
+  public static final SelenideElement          ELEMENT_EDIT_MY_PROFILE_BUTTON                                    =
+                                                                     $(byClassName("changeAvatar"));
```

Comment:
1-Don't modify an existed locator, it may impact other tests. if you can't reuse it, add a new one.
2-Wrong declaration of locator. ($(byClassName("changeAvatar")) is for changing avatar not edit profile).